### ISSUE TITLE
Missing sqlite3ext.h on bundle install

### ISSUE THREAD

**Issue by benjaminjackson** [westonplatter/phashion#44]
Here's the full output: https://gist.github.com/benjaminjackson/597ac027fb1a1e088fcd

I have sqlite3 installed and /usr/include/sqlite3ext.h exists.

**Comment by ypresto** [comment]
> I was missing both 'brew install libjpeg' and 'brew install libpng'

I installed libjpeg into ~/homebrew but this gem does not obey `--with-cppflags` and says could not find header file.

**Comment by MaxPleaner** [comment]
The readme says that Phasion's dependencies are installed through a tarball, but this doesn't seem to be the case (for me, running Ruby 2.4 on Ubuntu). I did fix the problem by installing libjpeg-dev and libpng-dev. I was initially confused, though, because the error I got when using `gem install phashion` was about a missing SQLite header.

**Comment by westonplatter** [comment]
@MaxPleaner yes, that is confusing. I'll check this out on a fresh ubuntu image this weekend. 

Maybe there needs to be a note in the readme about a libsqlite3-dev dependency for ubuntu machines.

**Comment by westonplatter** [comment]
@benjaminjackson @MaxPleaner after releasing version `1.2.0`,
- spun up a ubuntu 14.04 instance
- confirmed I could not install phashsion before installing `libjpeg-dev` and `libpng-dev`
- installed `libjpeg-dev` and `libpng-dev`
- successfully installed phashion

Could you retry with version `1.2.0`?

**Comment by westonplatter** [comment]
@MaxPleaner are you still having this issue?

**Comment by nebulousdog** [comment]
@westonplatter `libjpeg-dev` resolved similarly for me. also installed `libsqlite3-dev` and `sqlite3`.

**Comment by westonplatter** [comment]
@nebulousdog thanks for the input. Which OS are you on? Which package resolved the issue for you, `libjpeg-dev`, `libsqlite3-dev`, `sqlite3`? The reason for all the questions is that I'd like to update the readme with changes if any are needed.

**Comment by nebulousdog** [comment]
sure dude! I think `libjpeg-dev`/`libsqlite3-dev` fixed, and `sqlite3` was extra..

```
$ lsb_release -a 
No LSB modules are available. 
Distributor ID: Ubuntu 
Description: Ubuntu 16.04.3 LTS 
Release: 16.04 
Codename: xenial
 ```

**Comment by Nakilon** [comment]
How do `libjpeg-dev` and `libpng-dev` relate to SQLite? How do they manage to fix this error? And won't it be better to figure out why the flags are being ignored?

**Comment by Nakilon** [comment]
@MatzFan  How do `libjpeg-dev` and `libpng-dev` relate to SQLite?

**Comment by MatzFan** [comment]
@Nakilon happy to reopen if anyone provides a reproducible error with all the dependencies installed.
